rosidl_generator_py/rosidl_generator_py/generate_py_impl.py

```diff
@@ -97,16 +99,20 @@ def generate_py(generator_arguments_file, typesupport_impls):
     for module in modules:
         with open(os.path.join(args['output_dir'], module, '__init__.py'), 'w') as f:
             for module_, type_ in modules[module]:
-                f.write('from %s.msg._%s import %s\n' % (args['package_name'], module_, type_))
+                if type_.find('Request') == -1 and type_.find('Response') == -1:
```

Comment:
This condition is likely matching more cases then just request and response class since the substring can be anywhere.